.github/workflows/merge-pr.yml

```diff
@@ -12,7 +12,6 @@ jobs:
   changes:
     name: Determine Changed Files
     runs-on: ubuntu-22.04
-    container: ghcr.io/libertydsnp/frequency/ci-base-image:latest
```

Comment:
You'll see a lot of places where I removed the use of the ci-base-image. It really is only needed for things that use rust.